src/pages/faq.tsx

```diff
@@ -799,7 +141,7 @@ function FAQ() {
         </script>
       </Head>
 
-      <FAQWrapper>
+        
         <TopSection>
```

Comment:
TopSection is not necessary

Reply:
Use Section with fluid width, even if you want to have the curve, the css should only adjust that much and maybe call it HeroSection, though this should also have Content inside